.github/workflows/ignore_paths.yml

```diff
@@ -15,24 +15,55 @@ jobs:
           - ubuntu-latest
           - macos-latest
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-
-    - name: Run ShellCheck
-      uses: ./
-      id: check
-      with:
-        ignore_paths: ignore
-
-    - name: Verify check
-      run: | 
-        expect="testfiles/test.bash"
-        notexpect="testfiles/ignore/ignore.bash"
```

Comment:
There was no file at `testfiles/ignore/ignore.bash`, so it appears this test was trivially passing.